Sim/SimPapas/options/simple_papastool_CMS.py

```diff
@@ -0,0 +1,108 @@
+## Simple Papas Run for CMS
+## Runs papas using as a sequence of tools
+## The reconstructed particles are written to a ROOT file
+
+#
+#  To run PapasTools
+#  > ./run gaudirun.py Sim/SimPapas/options/simple_papastool_CMS.py
+#
+
+from Gaudi.Configuration import *
+from Configurables import ApplicationMgr, FCCDataSvc, PodioOutput
+from GaudiKernel import SystemOfUnits as units
+
+## read in generated particles from ROOT via podio
+#podioevent   = FCCDataSvc("EventDataSvc", input="./ee_ZH_Zmumu_Hbb.root")
+podioevent   = FCCDataSvc("EventDataSvc", input="./ee_Z_bbbar.root")
+
+from Configurables import PodioInput, ReadTestConsumer
+podioinput = PodioInput("PodioReader", collections=["GenVertex", "GenParticle"], OutputLevel=DEBUG)
+
+from Configurables import CMSFieldSvc, CMSTrackerSvc, CMSEcalSvc, CMSHcalSvc
```

Comment:
Is there a way to put all the papas thing in a specific configuration file that would then be imported in higher level configuration files? 

The user should be able to do: 

import papasalg

possibly change stuff like the detectors

add papasalg to the list of algorithms, as done at the very end of this file.

Reply:
One could have one papas configuration file per detector, 
but even then, there are things in common, and these common things should be organized in a common module somehow

Reply:
I have now managed to do this with a little help from Anna who pointed out an example of Valentin!

Reply:
thanks @zaborowska @vvolkl !